pages/community/events/community-call.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'AsyncAPI Community Call'
+type: COMMUNITY CALL
+description: 'Join us for our upcoming community call to see what’s on our roadmap, early feature demos and product updates from our engineering team.
+'
+weight: 1
+cover: https://cdn.dribbble.com/users/499731/screenshots/6648311/viceland_assets.png
+date: 2022-10-04T06:00:00+01:00
+---
+
+Join us for our upcoming community call to see what’s on our roadmap, early feature demos and product updates from our engineering team.
+
+### What you will learn
```

Comment:
```suggestion

```

hmm nope, this header doesn't work... i'd rather remove it unless you have a specific goal for a new section there